go/vt/vttablet/tabletmanager/vdiff/controller.go

```diff
@@ -104,9 +104,6 @@ func newController(ctx context.Context, row sqltypes.RowNamedValues, dbClientFac
 		TableDiffRowCounts:    stats.NewCountersWithSingleLabel("", "", "Rows"),
 		TableDiffPhaseTimings: stats.NewTimings("", "", "", "TablePhase"),
 	}
-	ctx, ct.cancel = context.WithCancel(ctx)
```

Comment:
Nit, but the `ctx` parameter is now unused in `newController()` so we could remove it.